rocket_component.py

```diff
@@ -0,0 +1,102 @@
+from atmosphere import Atmosphere
+import math
+
+
+class Rocket:
+  """
+  The rocket class will calculate the forces acting upon the rocket component
+  at a given time. Units will be metric for consistency.
+  """
+
+  def __init__(self) -> None:
+
+    # Basic Properties (Falcon 9 ex.)
+    self.altitude = 0  # m
+    self.length = 70  # m
+    self.width = 4  # m
+    self.velocity = 1000  # m/s
+
+    # Mass Properties (Falcon 9 ex.)
+    self.mass_fuel = 375000  # kg
+    self.mass_flow_rate = 1450  # kg/s
```

Comment:
if this is a flow rate for fuel, we should specify
```suggestion
    self.fuel_flow_rate = 1450  # kg/s
```

Reply:
fuel mass is the only thing changing. mathematically it makes sense as it would be represented as dm (change in mass)

Reply:
right, but if we have mass flow rate, the question becomes, mass flow rate of what?  WE know what to expect, but it could be a more meaningful name imo.